packages/iov-bns-governance/src/governor.spec.ts

```diff
@@ -0,0 +1,251 @@
+import { ReadonlyDate } from "readonly-date";
+
+import { Address, Identity, TokenTicker } from "@iov/bcp";
+import { bnsCodec, BnsConnection, VoteOption } from "@iov/bns";
+import { Ed25519HdWallet, HdPaths, UserProfile } from "@iov/keycontrol";
+
+import { COMMITTEE_IDS, Governor } from "./governor";
+import { CommitteeId, ProposalType } from "./proposals";
+
+function pendingWithoutBnsd(): void {
+  if (!process.env.BNSD_ENABLED) {
+    pending("Set BNSD_ENABLED to enable bnsd-based tests");
+  }
+}
+
+// The first IOV key (m/44'/234'/0') generated from this mnemonic produces the address
+// tiov15nuhg3l8ma2mdmcdvgy7hme20v3xy5mkxcezea (bech32) / a4f97447e7df55b6ef0d6209ebef2a7b22625376 (hex).
+// This account has money in the genesis file (see scripts/bnsd/README.md).
+const faucetMnemonic = "degree tackle suggest window test behind mesh extra cover prepare oak script";
+const faucetPath = HdPaths.iov(0);
+const bnsdUrl = "http://localhost:23456";
+
+async function getConnectionAndIdentity(
+  path = faucetPath,
+): Promise<{ readonly connection: BnsConnection; readonly identity: Identity }> {
+  const connection = await BnsConnection.establish(bnsdUrl);
+  const chainId = await connection.chainId();
+  const profile = new UserProfile();
+  const wallet = profile.addWallet(Ed25519HdWallet.fromMnemonic(faucetMnemonic));
+  const identity = await profile.createIdentity(wallet.id, chainId, path);
+  return {
+    connection: connection,
+    identity: identity,
+  };
+}
+
+describe("Governor", () => {
+  // TODO: Keys for COMMITTEE_IDS to be finalised
+  it("exports COMMITTEE_IDS constants", async () => {
+    expect(typeof COMMITTEE_IDS.testnet.GOVERNING_BOARD).toBe("number");
+    expect(typeof COMMITTEE_IDS.testnet.ECONOMIC_COMMITTEE).toBe("number");
+    expect(typeof COMMITTEE_IDS.testnet.VALIDATOR_COMMITTEE).toBe("number");
+    expect(typeof COMMITTEE_IDS.testnet.TECH_COMMITTEE).toBe("number");
```

Comment:
Can we use `toEqual` consistently when we want data equality and reserve `toBe` for object identity? I think it better expresses the intent and it is to easy to change an argument from string/number to an object, making the test wrong.